packages/react-components/src/components/VideoGallery/utils/videoGalleryLayoutUtils.test.tsx

```diff
@@ -320,15 +320,10 @@ const setup = (
     return null;
   };
 
-  let root;
-  act(() => {
-    root = create(<TestComponent {...initialArgs} />);
-  });
+  const { rerender } = render(<TestComponent {...initialArgs} />);
 
   if (updatedArgs) {
-    act(() => {
-      root.update(<TestComponent {...updatedArgs} />);
-    });
+    rerender(<TestComponent {...updatedArgs} />);
```

Comment:
This is the new format from new react test package?

Reply:
Sry the render one

Reply:
Ohh, you mentioned react 18 change its way to render components

Reply:
yeah

Reply:
Yeah newer syntax with the newer testing library. the render blocks don't need to be in act blocks